packages/DropdownMenu/src/index.tsx

```diff
@@ -1,31 +1,46 @@
 import React from 'react'
 import * as Ariakit from '@ariakit/react'
 import { CreateWuiProps, forwardRef, WuiProps } from '@welcome-ui/system'
+import { useTheme } from '@xstyled/styled-components'
+import type { WuiTheme } from '@welcome-ui/core'
 
 import { Arrow } from './Arrow'
 import { Item } from './Item'
 import { Separator } from './Separator'
 import * as S from './styles'
 
-export interface DropdownMenuOptions extends Ariakit.MenuProps {
+export interface DropdownMenuOptions extends Omit<Ariakit.MenuProps, 'gutter'> {
   /** add custom props from styled system on DropdownMenu inner */
   innerProps?: WuiProps
+  gutter?: keyof WuiTheme['space'] | number
 }
 
 export type DropdownMenuProps = CreateWuiProps<'div', DropdownMenuOptions>
 
 const DropdownMenuComponent = forwardRef<'div', DropdownMenuProps>(
-  ({ children, dataTestId, innerProps = {}, store, gutter = 10, ...rest }, ref) => {
+  ({ children, dataTestId, innerProps = {}, store, gutter = 'xs', ...rest }, ref) => {
+    const theme = useTheme()
     const arrowElement = store.useState('arrowElement')
     const isOpen = store.useState('open')
 
+    let parsedGutter = gutter
+    if (typeof parsedGutter === 'string') {
```

Comment:
Add test for this cases please

Reply:
Done and keen to get your feedback on [the commit](https://github.com/WTTJ/welcome-ui/pull/2205/commits/4b06a1c0c9abe81f8b8d0e78ab33988cf9c80c3f#diff-355b56cfd7481bba9c9332e59b81e99caa4794311322945ac7cd1e8724820b36R27)